addons/website_sale/tests/test_website_sale_cart.py

```diff
@@ -378,3 +378,20 @@ def test_keep_note_line(self):
             self.assertRecordValues(order.order_line, [{
                 "display_type": "line_note",
             }])
+
+    def test_checkout_no_delivery_available(self):
+        self.portal_user = self.user_portal
+        website = self.website.with_user(self.portal_user)
+        self.portal_user.write(self.dummy_partner_address_values)
+        self.carrier.country_ids = [2]
```

Comment:
Use `Command.set`

Reply:
I am not sure which part would be fit to use Comand.set